esprima.js

```diff
@@ -11,6 +11,11 @@
 var esprima = require('esprima-fb');
 var estraverse = require('estraverse-fb');
 var escodegen = require('escodegen');
+var listver = require('listver');
+
+var webpackMajorVersion = parseInt(
```

Comment:
@trusktr if you update this to use `loaderContext.version` and remove `listver` I’ll land and publish.